### PR TITLE
Phase zero graph

### PR DESCRIPTION
**Pull request by willgearty**
This replaces the number and list of usernames on the phase zero management page with a bigboard-esque graph of lottery registration over time.

Fixes https://github.com/learning-unlimited/ESP-Website/issues/2715.

Here's a picture of the graph on the cloud site:
image

### CODE REVIEW

File: esp/esp/program/modules/handlers/studentregphasezeromanage.py [hunk 1]
```diff
@@ -106,8 +107,12 @@ def phasezero(self, request, tl, one, two, module, extra, prog):
         context['role'] = role
         q_phasezero = Q(phasezerorecord__program=self.program)
         entrants = ESPUser.objects.filter(q_phasezero).distinct()
-        context['entrants'] = entrants
-        context['nentrants'] = len(entrants)
+
+        recs = PhaseZeroRecord.objects.filter(program=prog)
+        timess = [("number of lottery students", [(rec.user.count(), rec.time) for rec in recs])]
```
Comment (hwatheod) [inline esp/esp/program/modules/handlers/studentregphasezeromanage.py:112]: Probably I'm just misunderstanding something, but what does this `.count()` do ?  `rec.user` is already referring to a single user so what does it mean to count?

Comment (willgearty) [reply esp/esp/program/modules/handlers/studentregphasezeromanage.py:112]: `user` is actually a ManyToMany field in this case (I guess in hindsight, `users` would have been better):
https://github.com/learning-unlimited/ESP-Website/blob/4bc9787dc344d7ea15974ad9386ea98e89fca5d9/esp/esp/program/models/__init__.py#L2052-L2056
So `rec.user` is a set of users and `count()` is just counting how many users are in each `PhaseZeroRecord`.

Comment (willgearty) [reply esp/esp/program/modules/handlers/studentregphasezeromanage.py:112]: Googling around it appears `rec.user.all().count()` may also work, but I think `.count()` will either be as efficient or more efficient..

Comment (hwatheod) [reply esp/esp/program/modules/handlers/studentregphasezeromanage.py:112]: Okay, I get it now.  The `user` as opposed to `users` threw me off.

**Pull request merged**